src/init/features/project.spec.ts

```diff
@@ -94,26 +96,6 @@ describe("project", () => {
           displayName: "my-project",
         });
       });
-
-      it("should throw if project ID is empty after prompt", async () => {
-        const options = {};
-        const setup = { config: {}, rcfile: {} };
-        prompt.select.onFirstCall().resolves("Create a new project");
-        prompt.input.resolves("");
-        configstoreSetStub.onFirstCall().resolves();
-
-        let err;
-        try {
-          await doSetup(setup, emptyConfig, options);
-        } catch (e: any) {
-          err = e;
-        }
-
-        expect(err.message).to.equal("Project ID cannot be empty");
-        expect(prompt.select).to.be.calledOnce;
-        expect(prompt.input).to.be.calledTwice;
-        expect(createFirebaseProjectStub).to.be.not.called;
-      });
```

Comment:
I moved the check into the prompt itself.

This works in a better way now. Disallow enter to hit when ID is invalid.

<img width="1717" height="411" alt="Image" src="https://github.com/user-attachments/assets/fa5053df-9cfe-405d-8a7e-94eb82427b77" />